### ISSUE TITLE
I think FSRS has been predicting intervals of my previous SM-2 reviews too far away

### ISSUE THREAD

**Issue by QuietBasil** [open-spaced-repetition/fsrs4anki#582]
I think FSRS has been predicting intervals of my previous SM-2 reviews too far away and probably also my newer FSRS cards too. And now I feel like my anki is in ruins and no longer being helpful for my studies.

Also, right after switching, I had only rescheduled "cards reviewed in the last 7 days" for once and then never rescheduled again - although just doing this significantly decreased my future daily reviews to about 70-75% of original in the stats

My recall has drastically and suddenly changed from 94% to 75% with drastically decreased recalling speed from around 13s to about 20s per card.

I even tried manually changing the retention from 90% to 93% in the FSRS settings but it still didn’t help.

Also as far as I recall, I used the buttons in SM-2 in a good way and am also using them in quiet the same way for FSRS (pressing good when I got the answer, hard when I thought it was difficult but correct, again when I couldn't recall or got it wrong and pressed easy when I felt it was really easy for me)

Also, I used the same settings for my decks as the Anking showed on Youtube previously for SM-2 as well as for the updated new settings for FSRS. Although in FSRS, I did make a minor change in the relearning steps of lapses from "15m" to "30m" (because I feel I am not actually relearning the lapsed card well before sending it back to a far-off review date and do still feel the same).

Even though I am scared I will keep getting many cards wrong, still keep going everyday (since more than a month now) with terrible recall and frustration hoping that the smarter FSRS algorithm will someday turn things around and everything will get better soon but nothing is changing and apparently is getting worse. I am very worried because I had made anki my primary study material and now I feel like I have ruined it.

I would really appreciate it if any expert (maybe [u/LMSherlock](https://www.reddit.com/u/LMSherlock/) or [u/ClarityInMadness](https://www.reddit.com/u/ClarityInMadness/) or somebody involved in developing FSRS) or anybody could provide any solution because I would like to still reap the benefits of spaced repetition in my life.

Also I would really appreciate your experiences with changing to FSRS especially when you had already done a massive amount of cards on the previous SM-2.

Below are pictures of my settings and stats if you need.

![Screenshot (7167)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/ae1ba7f0-4884-4116-bd12-a0b4b60fd020)

![Screenshot (7168)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/32f6d93f-0e31-4886-9e0c-00fee386b7b1)
![Screenshot (7170)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/4167d167-76f7-4ce9-a356-91a32093bb4a)
![Screenshot (7171)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/73acb6d7-52da-4436-be3e-d6e15d9c43c4)
![Screenshot (7172)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/b2dcb5a9-c6aa-4a3e-a670-6f80261146c9)
![Screenshot (7173)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/99897b6c-4e1c-4bf7-877d-5bf437872355)
![Screenshot (7174)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/078fddbb-a5b3-44a0-80e5-43f8e7c7aa90)
![Screenshot (7175)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/249849ca-39ef-4017-b436-b3734c226ad1)
![Screenshot (7176)](https://github.com/open-spaced-repetition/fsrs4anki/assets/154599999/daee8370-5b69-44ec-971a-a6a0c64736e5)

**Comment by L-M-Sherlock** [comment]
![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/32575846/609a559a-f777-4e31-9869-7edf07a88f03)

Your retention has tended to decrease since six weeks ago. Do you remember what did you do in six weeks ago?

**Comment by L-M-Sherlock** [comment]
![calibration_0](https://github.com/open-spaced-repetition/fsrs4anki/assets/32575846/777b35cf-4208-441c-9f42-51005cea0248)

R-squared: 0.8585
RMSE: 0.0152
MAE: 0.0100

The calibration and metrics show that the prediction of FSRS is accurate in high retention area. I recommend increasing your retention to 0.95 and reschedule all your cards.

**Comment by QuietBasil** [comment]
I don't understand the graphs. But I would really appreciate any solution that would be helpful in the long term so that I may do well on my upcoming exam USMLE step 2 in 4-5 months and also further exams in residency and so on. I cannot risk experimentations any further because that may potentially cause me a lot of trouble and mental breakdowns.

**Comment by QuietBasil** [comment]
Would going with your new recommended settings be better than simply switching back to SM-2?

**Comment by QuietBasil** [comment]
> ![image](https://private-user-images.githubusercontent.com/32575846/295844034-609a559a-f777-4e31-9869-7edf07a88f03.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5NjQ0MzAsIm5iZiI6MTcwNDk2NDEzMCwicGF0aCI6Ii8zMjU3NTg0Ni8yOTU4NDQwMzQtNjA5YTU1OWEtZjc3Ny00ZTMxLTk4NjktN2VkZjA3YTg4ZjAzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTExVDA5MDg1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3YWM4NGFhNzFmYjljYzlkNzlkYzJhNzVjMTQ4MmIxNzM2Y2ZmODk1YTE3ZjlmYjI2ZDIyNDdkNzRkNTdmYzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.npkiDeK1ClB-C6pe0Agko9G4t7FG8mIVUG8-irOthpI)
> 
> Your retention has tended to decrease since six weeks ago. Do you remember what did you do in six weeks ago?

Yes six weeks ago I switched to FSRS and I used to FSRS helper addon to reschedule cards studied in the last 7 days - and since then everything has changed

**Comment by gugutu** [comment]
I have encountered a similar problem. When I used the default parameters of FSRS, it worked very well. After optimizing the parameters (indicated by the arrow in the image), the number of reviews it arranged for me decreased significantly. However, a few days later, I found that the retention rate sharply decreased (the expected retention rate I set was 0.9), which made me give up for a period of time. Now I have changed back to using the default parameters.

![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/59798823/ee3f6380-6e2d-4ca8-a34d-a616b595b971)
